routers/api/v1/utils/hook.go

```diff
@@ -21,6 +21,37 @@ import (
 	webhook_service "code.gitea.io/gitea/services/webhook"
 )
 
+// getBoolFromMap extracts a boolean value from a map with a default fallback
+//
+//nolint:unparam // defaultValue is needed for generic helper function
+func getBoolFromMap(m map[string]any, defaultValue bool) bool {
+	if val, ok := m["enable"]; ok {
+		if boolVal, ok := val.(bool); ok {
+			return boolVal
+		}
+	}
+	return defaultValue
+}
+
+// getIntFromMap extracts an integer value from a map with a default fallback
+//
+//nolint:unparam // defaultValue is needed for generic helper function
+func getIntFromMap(m map[string]any, defaultValue int) int {
+	if val, ok := m["limit"]; ok {
+		switch v := val.(type) {
+		case int:
+			return v
+		case float64:
+			return int(v)
+		case string:
+			if intVal, err := strconv.Atoi(v); err == nil {
+				return intVal
+			}
+		}
+	}
+	return defaultValue
+}
+
```

Comment:
These functions have oddly generic names but then retrieve hardcoded strings "enable" and "limit" from the map. They should be renamed to like `getEnableInt` and `getLimitInt`. Also I wonder if the nolint could be avoided.